connections/elastic_models.py

```diff
@@ -79,8 +79,8 @@ class Apartment(ReadOnlyDocument):
     estate_agent_email = Keyword()
     estate_agent_phone = Keyword()
 
-    coordinate_lat = Long()
-    coordinate_lon = Long()
+    project_coordinate_lat = Long()
+    project_coordinate_lon = Long()
```

Comment:
Should these be `Float` instead of `Long`? The ES index has defined coordinates as floats.

Reply:
Ah, these have been changed at some point. I'll change.